src/site/README.md

```diff
@@ -307,3 +307,36 @@ The following query looks up the `entityId` for one page of the `entityBundle` (
 ```
 
 </details>
+
+## Testing
```

Comment:
@holdenhinkle Any reason Cypress wasn't used for these tests?

Reply:
Hey @timwright12 , I just added a 'why not cypress' note to the documentation.